sasmodels/multiscat.py

```diff
@@ -472,7 +472,7 @@ def __init__(self, qmin=None, qmax=None, nq=None, window=2,
             # 1-D radial profile is determined by the q values we need to
             # compute, either for the calculated q values for the resolution
             # function (if any) or for the raw q values desired
-            self._q = np.linspace(qmin, qmax, nq//(2*window))
+            self._q = np.linspace(qmin, qmax, nq//int(2*window))
```

Comment:
I don't remember what the code is doing so I don't know what `self._q` should be. My guess is that older numpy implicitly wrapped and floating point lengths with `int`, so `int(nq//(2*window))` is a safer choice.